utilities/PerformanceTesting/query.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
```

Comment:
I hate to suggest more large changes, but this is very complicated for what it's doing so I wonder whether it might make more sense as a python script. The json construction will be easier, the queries should be easier too, and generally it should be a lot easier to follow and understand.